cloud-openstack-server/src/main/java/jetbrains/buildServer/clouds/openstack/OpenstackCloudImage.java

```diff
@@ -47,14 +50,16 @@
     @Nullable
     private final CloudErrorInfo errorInfo;
 
-    public OpenstackCloudImage(@NotNull final String imageId, @NotNull final String imageName, @NotNull final OpenstackApi openstackApi,
-            @NotNull final String openstackImageName, @NotNull final String flavorId, @NotNull final CreateServerOptions options,
-            @Nullable final String userScriptPath, @NotNull final ServerPaths serverPaths, @NotNull final ScheduledExecutorService executor) {
+    public OpenstackCloudImage(@NotNull final OpenstackApi openstackApi, @NotNull final String imageId, @NotNull final String imageName,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 10 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/axel3rd-github/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jetbrains.buildServer.clouds%3Acloud-openstack&pullRequest=30&issues=AWTrjQiwTfAogIny1gBu&open=AWTrjQiwTfAogIny1gBu)

[ISSUE_KEY]: AWTrjQiwTfAogIny1gBu